projects/shared/src/lib/pipe/identifiedby-label.pipe.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2023 RERO
+ * Copyright (C) 2023 UCLouvain
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'identifiedByLabel'
+})
+export class IdentifiedByLabelPipe implements PipeTransform {
+
+  transform(identifiedBy: any[], types: string[]): string | null {
+    const identifiers: Array<any> = types ? identifiedBy
+      .filter(identifier => types.includes(identifier.type)) : identifiedBy;
+    return identifiers ? identifiers.map(data =>
+      data.value
+    ).join(', ') : null
```

Comment:
cosmetic 
```typescript
const identifiers: Array<any> = types 
    ? identifiedBy.filter(identifier => types.includes(identifier.type)) 
    : identifiedBy;
return identifiers 
    ? identifiers.map(data => data.value).join(', ') 
    : null
```
Maybe you could create a "glue" param (with a default value to ', ') and use it in the join
Missing a ";" at the end of return statement.